setup.py

```diff
@@ -24,11 +24,17 @@
     author_email="softapps@rigetti.com",
     description="A collection of quantum algorithms built using pyQuil and Forest",
     url="https://github.com/rigetticomputing/grove.git",
-    packages=["grove", "grove.pyqaoa", "grove.pyvqe", "grove.qft",
-              "grove.phaseestimation", "grove.teleport",
-              "grove.deutsch_jozsa", "grove.arbitrary_state",
-              "grove.bernstein_vazirani", "grove.simon",
-              "grove.amplification"],
+    packages=["grove",
+              "grove.pyqaoa",
+              "grove.pyvqe",
+              "grove.qft",
+              "grove.alpha.phaseestimation",
```

Comment:
Do you want `grove.alpha` here?

Reply:
You're also missing `fermion_transforms` (although I see it wasn't there to begin with) in this list.